test/libsolidity/syntaxTests/types/array_index_too_large.sol

```diff
@@ -0,0 +1,7 @@
+contract C {
+    function f() public returns (string memory) {
+        return (["zeppelin"][123456789012345678901234567890123456789012345678901234567890123456789012345678]);
```

Comment:
Ah, perhaps add a comment `// this used to cause an internal error` so that we know why the test is there.